src/crl.rs

```diff
@@ -135,8 +137,24 @@ impl<'a> TryFrom<&'a [u8]> for CertRevocationList<'a> {
     ///
     /// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
     fn try_from(crl_der: &'a [u8]) -> Result<Self, Self::Error> {
+        CertRevocationList::new(crl_der, der::TWO_BYTE_DER_SIZE)
```

Comment:
I think the `try_from` form should be able to parse the largest CRL someone could encounter (so, >16MB). The limitation to 64KB here seems surprising and not really addressing a real DOS issue?

Reply:
> I think the try_from form should be able to parse the largest CRL someone could encounter (so, >16MB).

:+1: In the corpus of CRLs I scraped from CCADB the largest was 56mb. 

We could either set a limit based on that data, or just use the limit for what we can support parsing, the four-byte long-form length boundary.

Do you have a preference?

Reply:
I think I prefer the latter for CRLs.

Reply:
Reworked to use the maximum supported size.